graphdatascience/query_runner/query_runner.py

```diff
@@ -7,9 +8,65 @@
 from .graph_constructor import GraphConstructor
 
 
+class EndpointType(Enum):
+    PROCEDURE = "PROCEDURE"
+    FUNCTION = "FUNCTION"
+
+
 class QueryRunner(ABC):
+    def call_function(
```

Comment:
(nitpick) Might be better to have a bit more elaborate name, since it sounds like it's just calling a Python function